azafea/event_processors/endless/metrics/v3/model/_base.py

```diff
@@ -70,23 +70,63 @@
 class Channel(Base):
     __tablename__ = 'channel_v3'
 
-    __table_args__ = (
-        UniqueConstraint('image_id', 'site', 'dual_boot', 'live'),
-    )
-
```

Comment:
Why would we remove this constraint? It should still be guaranteed that there is only a single row for each combination of these flags.

Reply:
I did not remove it, I moved it further down with the declaration of indexes on the site_*  (lines 98-112 )

Reply:
Ah, I see it, missed that!